src/main/java/frc/robot/Commands/DropPlacers.java

```diff
@@ -0,0 +1,16 @@
+package frc.robot.Commands;
+
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import frc.robot.Subsystems.ClimberPlacer;
+
+public class DropPlacers extends ParallelCommandGroup {
```

Comment:
Maybe change this to `DropBothPlacers` of something if you've already have dropPlacer